ignite/handlers/time_limit.py

```diff
@@ -0,0 +1,42 @@
+import logging
+import time
+from typing import Optional
+
+from ignite.engine import Engine
+
+log = logging.getLogger(__name__)
```

Comment:
Currently, we create an instance of the logger inside the class, for example: https://github.com/pytorch/ignite/blob/c25ce51dac5a0d519362449f6b6af4a6f82e9bba/ignite/handlers/early_stopping.py#L79

Let's do it like that here too.